core/store/orm/orm.go

```diff
@@ -1265,9 +1265,14 @@ func (orm *ORM) KeyExists(address []byte) (bool, error) {
 	return true, err
 }
 
+// ArchiveKey soft-deletes a key whose address matches the supplied bytes.
+func (orm *ORM) ArchiveKey(address []byte) error {
+	return orm.DB.Where("address = ?", address).Delete(models.Key{}).Error
+}
+
 // DeleteKey deletes a key whose address matches the supplied bytes.
 func (orm *ORM) DeleteKey(address []byte) error {
```

Comment:
Isn't this method now identical to the one above?

Reply:
Nah, `DeleteKey` uses `Unscoped`